composeApp/src/desktopMain/kotlin/org/ooni/probe/data/DeepLinkHandler.kt

```diff
@@ -0,0 +1,278 @@
+package org.ooni.probe.data
+
+import co.touchlab.kermit.Logger
+import org.ooni.probe.APP_ID
+import tk.pratanumandal.unique4j.Unique4j
+import tk.pratanumandal.unique4j.exception.Unique4jException
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.Date
+
+class DeepLinkHandler {
```

Comment:
I feel we could simplify this class by splitting it into the 3 things it's trying to do:
- Make sure only one app instance is running, and passing the args of every open to the first instance
- Parsing deeplinks
- Registering protocol handlers